src/app/base/hooks/node.ts

```diff
@@ -25,6 +25,7 @@ export type MachineMenuAction = Exclude<
   | NodeActions.SET_ZONE
   | NodeActions.TAG
   | NodeActions.UNTAG
+  | NodeActions.CHECK_POWER
```

Comment:
I think these were arranged somewhat alphabetically, perhaps you could take it to the top

Reply:
Sure thing, will do